apps/io-wallet-user-func/src/app/config.ts

```diff
@@ -50,6 +51,50 @@ export const ANDROID_CRL_URL =
 export const ANDROID_PLAY_INTEGRITY_URL =
   "https://www.googleapis.com/auth/playintegrity";
 
+export const MailConfig = t.type({
+  mailFeatureFlag: t.boolean,
+  mailSender: NonEmptyString,
+  mailupSecret: NonEmptyString,
+  mailupUsername: NonEmptyString,
+});
+
+export type MailConfig = t.TypeOf<typeof MailConfig>;
+
+export const WALLET_ACTIVATION_EMAIL_SUBJECT =
+  "IT Wallet - Aggiungi i tuoi documenti al Portafoglio di IO";
+export const WALLET_ACTIVATION_EMAIL_TEXT =
+  "IT Wallet - Aggiungi i tuoi documenti al Portafoglio di IO";
+export const WALLET_ACTIVATION_EMAIL_FAQ_LINK =
+  "https://io.italia.it/documenti-su-io/faq/#n1_12";
```

Comment:
Why the anchor to a specific FAQ?

Reply:
I think it opens directly to the FAQ "Quali sono i requisiti tecnici minimi per attivare Documenti su IO?", which is a topic of the email

Reply:
Is this functionality required? Or it should simply open the list of all faq. I don't see the reason for why open the FAQ that reports the compatibility problems.

Reply:
Where did you see that the topic of the email are the minimum requirements?